pyscriptjs/src/main.ts

```diff
@@ -225,13 +238,12 @@ from pyscript import micropip, Element, console, document`);
 
         logger.info('Packages to install: ', this.config.packages);
         await runtime.installPackage(this.config.packages);
-        await this.fetchPaths(runtime);
 
         // Finally load plugins
         await this.fetchPythonPlugins(runtime);
     }
 
-    async fetchPaths(runtime: Runtime) {
+    fetchPaths(runtime: Runtime) {
         // XXX this can be VASTLY improved: for each path we need to fetch a
```

Comment:
We can probably remove this comment since we aren't using `loadFromFile` anymore and fetching things with JS 😄